src/operator/nn/mkldnn/mkldnn_rnn_impl.h

```diff
@@ -118,7 +151,7 @@ static size_t GetMKLDNNRNNCacheMemorySize(int L,
       break;
     case rnn_enum::kGru:
       size = 2 * (D * (I + H) * 3 * H + (L - 1) * D * (D * H + H) * 3 * H +
-             L * D * 2 * N * H) + T * N * D * H + L * 2 * D * 3 * H + (L + 2) * D * 2 * N * H +
+             L * D * 2 * N * H) + T * N * D * H + L * 2 * D * 4 * H + (L + 2) * D * 2 * N * H +
```

Comment:
Nit: I know it's out of the scope of this PR, but could we rename the variables to something more self-explanatory?

Reply:
No problem. I will have a try. I ever did it. But it seems that there is a large amount of code. I think we can segregate the common parts from them.

Reply:
@marcoabreu Sorry for the late update. I have renamed this part. Would you mind checking it again? Thanks.